### ISSUE TITLE
Still randomly crashes between loading zones

### ISSUE THREAD

**Issue by 03bgood** [gamestabled/OoT3D_Randomizer#743]
I'm not sure what's causing it, but it happens with both the cartridge and digital version/CIA dump. Sometimes when going through loading zones, or even warping; It just randomly (very rarely) boots me out of the game and says "an unknown error had occurred". I don't have "exception handlers" enabled, but this issue STILL hasn't been fixed. Have no idea what is causing it. I have to constantly save every 2-5 minutes. It's so annoying and I have no idea what could be causing it. Again, it's very rare and usually happens only once per seed.

**Comment by PhlexPlexico** [comment]
Closing as duplicate of https://github.com/gamestabled/OoT3D_Randomizer/issues/529

As mentioned in previous issues, please enable Exception Handlers in Luma and provide the crash dump here. If this is not done **we will not be able assist you**. Please also include the spoiler log, which is located at `sdcard:/OoT3DR/Logs`.

Thanks.
